src/aqua/installation-spell/src/aqua/deal_spell.aqua

```diff
@@ -1,108 +1,12 @@
-import Op, Debug, Hash, Dist, Srv, PeerId from "@fluencelabs/aqua-lib/builtin.aqua"
-import Spell from "@fluencelabs/spell/spell_service.aqua"
-import Registry from "@fluencelabs/registry/registry-service.aqua"
-import resolveResource from "@fluencelabs/registry/resources-api.aqua"
-import getInsecuredPeerId, createSubnetwork, registerWorker from "@fluencelabs/registry/subnetwork.aqua"
-
-import JsonStr, ModuleConfigJson from "./json.aqua"
-import Address, Multiaddr, CID, WorkerSpellArgs from "./types.aqua"
-import Console from "./console.aqua"
-import is_deployed, install from "./spell.aqua"
-import SpellData from "./spell_data.aqua"
+import Address, Multiaddr, CID from "./types.aqua"
+import install from "./spell.aqua"
 import log from "./log.aqua"
+import get_status from "./status.aqua"
 
-const DISCOVERY_TIMEOUT_SEC = 43200
-
-service Scope("scope"):
-    get_peer_id() -> PeerId
-
-func create_sub(deal_id: string) -> ?string, *string:
-    subnet_id, errors <- createSubnetwork(deal_id)
-    <- subnet_id, errors
-
-func resolve_workers(deal_id: Address):
-    on HOST_PEER_ID:
-        peer_id <- getInsecuredPeerId()
-        id <- Registry.get_key_id(deal_id, peer_id)
-        records, errors <- resolveResource(id, 1)
-
-    Console.print("records:")
-    for r <- records!:
-        Console.print(r.metadata.copy(issuer_signature = []))
-
-    Console.print("errors:")
-    Console.print(errors)
-
--- A hack to allow using timestamp as u32 values
--- Aqua doesn't allow truncating values
-service PeerTimeTrunc("peer"):
-    timestamp_sec() -> u32
 
 -- this is executed inside Deal Installation Spell
 func deal_install(ipfs: Multiaddr, worker_def_cid: CID, deal_id: Address):
-    spell_id <- SpellData.spell_id()
-    Spell spell_id
-
-    worker_spell: ?string
-    try:
-        worker_spell <- Srv.resolve_alias("worker-spell")
-    catch e:
-        worker_spell <<- spell_id
-
-        try:
-            -- if alias isn't set, then set it as early as possible
-            Srv.add_alias("worker-spell", spell_id)
-            log(["alias created 'worker-spell'", spell_id])
-        catch add_err:
-            log(["error creating alias", add_err])
-
-    if worker_spell! != spell_id:
-        log(["Another worker spell is deployed on this worker (existing, current)", worker_spell!, spell_id])
-    else:
-        log(["Installing worker for deal", deal_id])
-        install(ipfs, worker_def_cid)
-        status <- Spell.get_string("status")
-        log(["Worker installation finished with status", status.str])
-
-        if status.str == "install finished":
```

Comment:
why did you removed this logic? 

don't remember why we needed it, but I guess that's because of too many requests from registry

Reply:
I will update code for direct workers deploy, thx